Documentation/supermuc.rst

```diff
@@ -156,7 +157,8 @@ set compiler options:
   metis='yes'
   netcdfDir=os.environ['NETCDF_BASE']
   hdf5Dir=os.environ['HDF5_BASE']
-
+  metisDir=os.environ['PARMETIS_BASE']
```

Comment:
is this used by any script?

Reply:
There seems an update in module HDF6 and Metis on NG and some names are changed.

It seems that metisDir is needed in the compilation.